charts/argo-cd/values.yaml

```diff
@@ -579,6 +579,18 @@ controller:
   # Additional replicas will cause sharding of managed clusters across number of replicas.
   replicas: 1
```

Comment:
If I've understood docs correctly it seems it's possible to have graceful failover now. If number of replicas changes the shards will rebalance. Might be worth adding note about this.